Cargo.toml

```diff
@@ -24,4 +24,5 @@ core-foundation = "0.9.1"
 
 [dev-dependencies]
 gl = "0.14.0"
-winit = "0.22.2"
+winit = "0.24"
+sdl2 = { version = "0.34", features = ["raw-window-handle", "bundled"] }
```

Comment:
Bringing in SDL2 for the SDL2 example. Some options here:
 * Could remove the SDL2 example to avoid the dev-dependency (shouldn't affect downstream consumers of the crate)
 * The "bundled" feature increases build time but I find it more reliable to use. Without it, people building the crate will need to install/set up SDL2 which is a little annoying on windows.